server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -71,55 +87,59 @@ public ServerResource(InvocationHelper invocationHelper) {
    public Invocations getInvocations() {
       return new Invocations.Builder()
             .invocation().methods(GET).path("/v2/server/")
-               .handleWith(this::info)
+            .handleWith(this::info)
             .invocation().methods(GET).path("/v2/server/config")
-               .permission(AuthorizationPermission.ADMIN).auditContext(AuditContext.SERVER).handleWith(this::config)
+            .permission(AuthorizationPermission.ADMIN).auditContext(AuditContext.SERVER).handleWith(this::config)
+            .invocation().methods(GET).path("/v2/server/config").withAction("listen")
```

Comment:
I'm not sure `/v2/server/config` is the right endpoint for the cache configuratin SSE. Configuration events relate to things defined within the container (schemas and scripts are stored in a cache), shouldn't this be exposed via `/v2/cache-managers/{name}/config?action=listen`? Or on it's own endoint e.g .`/v2/config?action=listen`

@gustavonalle Any thoughts?

Reply:
Not fond of the  `cache-managers` name, because we don't really intend to support more than one and also because it implies *caches* whereas we want to be a bit more inclusive.

Reply:
Maybe we should deprecate `/v2/cache-managers/{name}` in favour of `/v2/container/`. That way it's clear we only have a single container and there is no association with caches. Looking at the exposed endpoints on `cache-managers`, the only ambigous endpoint would be `/v2/cache-managers/{name}/config` but we can change that to `/v2/container/global-config`.

So something like:

```
/v2/container/config?action=list
# Replacement for `/v2/cache-managers/{name}/config`
/v2/container/global-config

/v2/container/health
/v2/container/cache-configs
...
# Other existing `/v2/cache-managers/{name}` endpoints
```

Reply:
Yes, I like that. Let's get confirmation from @gustavonalle

Reply:
Formats in all REST endpoint is usually communicated through the ```Accept``` HTTP header, passing a MediaType

Reply:
> Formats in all REST endpoint are usually communicated through the `Accept` HTTP header, passing a MediaType

Argh of course :rofl:

Reply:
> Maybe we should deprecate /v2/cache-managers/{name} in favour of /v2/container/

Sounds good

Reply:
In fact the code does just that: it reads the `Accept` header and will send back configs in that format. Obviously the content-type of the SSE response will always be `text/event-stream`